Базовая стилизация всех страниц

Regeace opened the pull request:
В созданный файл стилей подключены шрифты, выполнена базовая стилизация страниц.

---
:mortar_board: [Базовая стилизация всех страниц](https://up.htmlacademy.ru/htmlcss/33/user/1812401/tasks/6)

:boom: https://htmlacademy-htmlcss.github.io/1812401-sedona-33/5/

Regeace committed:
Добавляет стили

В файл стилей подключены шрифты, выполнена базовая стилизация страниц.

LoremLipsum commented on styles/styles.css:
```diff
@@ -0,0 +1,298 @@
+}
+
+.navigation-item {
+  text-align: left;
```
Можно пока убрать. Скорее всего не понадобится

LoremLipsum commented on styles/styles.css:
```diff
@@ -0,0 +1,298 @@
+  text-align: left;
+}
+
+.page-header .button {
```
Для маленьких кнопок добавляют дополнительный класс `button-small`. Маленькие или большие кнопки, ил кнопки разных цветов не зависят от блока внутри которого они находятся и могут использоваться где угодно.

В разметке
`<a class="button button-small">Button</button>`
`<a class="button button-large">Button</button>`
`<a class="button button-main">Button</button>`
`<a class="button button-special">Button</button>`

В стилях
```
.button {
  // Общие стили для всех основных (прямоугольных) кнопок_
}

.button-small {
  // Стили которые переопределяют размеры для маленькой кнопки, например размер шрифта, паддинги ипр.
}

.button-large {
  // Стили которые переопределяют размеры для большой кнопки, например размер шрифта, паддинги ипр.
}

.button-main {
  // Кнопка с коричневым фоном
}

.button-special {
  // Кнопка с голубым фоном
}
```

LoremLipsum commented on styles/styles.css:
```diff
@@ -0,0 +1,298 @@
+
+.main-index-header {
+  width: 1200px;
+  height: 485px;
```
Старайся не ставить фиксированную высоту, используй `min-height`. Может не пройти тест на переполнение

LoremLipsum commented on styles/styles.css:
```diff
@@ -0,0 +1,298 @@
+  width: 1200px;
+  height: 485px;
+  background-color: #8f0d0d;
+  background-image: url("../images/index-background.jpg");
```
У фонов ещё надо прописывать
`background-position`
`background-size`
`background-repeat`

LoremLipsum commented on styles/styles.css:
```diff
@@ -0,0 +1,298 @@
+}
+
+.reasons-item:first-of-type,
+.reasons-item:nth-of-type(2) {
```
Лишкам с голубым фоном и картинками задай дополнительные классы.
Чаялды не совсем подходят, потому что по идее такую лишку можно поставить на любое место. Можно перенести их в конец списка, например

LoremLipsum commented on styles/styles.css:
```diff
@@ -0,0 +1,298 @@
+  background-color: #ffffff;
+}
+
+.advantages-item:first-of-type,
```
О здесь нужно использовать псевдоклассы с чаялдами. Но пропиши фоны для четных и нечетных элементов. Они чередуются. Сайчас ты указал для конкретных лишек. А лишки могут добавляться, убавляться

LoremLipsum commented on styles/styles.css:
```diff
@@ -0,0 +1,298 @@
+}
+
+.reasons-item-title,
+.advantages-title {
```
Не группируй свойства, Это усложняет внесение правко для отдельных элементов.
И на втором уровне будет БЭМ и вы будуте делить стили на отдельные блоки, а там точно так прописать не получится

LoremLipsum commented on styles/styles.css:
```diff
@@ -0,0 +1,298 @@
+  text-decoration: none;
+}
+
+.catalog-card-link:first-of-type {
```
Здесь тоже задай дополнительные классы для button, где я писала про цвет и размеры кнопок.
Через чаялды здесь точно стилизовать нельзя. Порядок элементов в карточке может поменяться

Regeace committed:
Вносит изменения в стили и классы

Исправлены и оптимизированы стили элементов, добавлены дополнительные классы элементам списков, кнопкам.

Regeace committed:
Исправляет классы

Исправлены классы кнопок у карточек каталога.
